domain/src/main/java/org/oppia/domain/profile/ProfileManagementController.kt

```diff
@@ -132,8 +133,12 @@ class ProfileManagementController @Inject constructor(
     avatarImagePath: Uri?,
     allowDownloadAccess: Boolean,
     colorRgb: Int,
-    isAdmin: Boolean
+    isAdmin: Boolean,
+    storyTextSize: Float?,
```

Comment:
It seems a bit unclean to use direct sp values for representing story text size. Could we use an enum, instead? Then, we can translate the enum to actual text size in one location (which ought to map to dimens values to keep dimension things outside of code). Consider using an enum defined in the proto file to simplify things.

Reply:
I tried creating enum for story_text_size in profile proto. I am not clear on how I can set value for it or how I can use it. Can you please check profile.proto whether its the right thing to do.
Also please suggest me how I can use this.

Reply:
This is what I had in mind—your code looks correct to me. When it comes to actually handling the zoom, we will likely have one location in code that switches across all of the constants and associates them with a multiplier that we use to scale all text in the app. Does that help?

Reply:
If defining of proto for story_text_size is correct. I am not understanding how I can make use of it in domain layer. I need to have floating values for text sizes. How I can set it for these enums.

Reply:
As we discussed on 1:1, I referred to https://github.com/oppia/oppia-android/blob/5b0d9cf11efe31e0d5b1fa6a391a5699db59995a/model/src/main/proto/thumbnail.proto#L19 and tried to implement similar way. PTAL.
I have created method `getStoryTextSize(storyTextSize: StoryTextSize)` in ProfileControlManagement that returns float values for storyTextSize.